src/BloomExe/Program.cs

```diff
@@ -269,6 +269,7 @@ static void Main(string[] args1)
 			}
 			finally
 			{
+				Palaso.UI.WindowsForms.Reporting.MemoryManagement.CheckMemory(true, "Bloom finished and exiting", false);
```

Comment:
Does this mean the user could see a warning (telling him he is short of memory and needs to restart Bloom) AFTER he clicked the box to close it altogether?